packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -59,6 +60,8 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
 
   List<FlameChartRow> rows = [];
 
+  List<FlameChartSection> sections = [];
```

Comment:
nit: look like this and maybe rows can be final as their values are manipulated by the actual List isn't reset.

Reply:
made vars final